content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -10,8 +10,233 @@ tags:
 
 {{< toc >}}
 
-
 {{< include file="/_includes/CLI/CLIGuideWIP.md" type="page" >}}
 
+The **group** namespace contains 8 commands and is based on functions found in the SCALE API and web UI. It provides access to group account creation, configuration, and management.
+
+You can enter commands from the main CLI prompt or from the system namespace prompt.
+
+### The create Command
+The `create` command creates a new group.
+
+{{< expand "Using the create Command" "v" >}}
+From the CLI prompt, enter:
```

Comment:
after the expand shortcode, provide more information/details about this command. the options, included, what is involved in using the command. We explain the command in more details. The reason we hide this behind the first expand is if we decide to drop this section into a non-CLI tutorial as a snippet (that includes the expands), that snippet has the information the reader needs in the tutorial.